source/plugins/wakatime/index.mjs

```diff
@@ -43,6 +26,9 @@ export default async function({login, q, imports, data, account}, {enabled = fal
     const {data: {data: stats}} = await imports.axios.get(`${url}/api/v1/users/${user}/stats/${range}?api_key=${token}`)
 
     const projectStats = stats.projects?.map(({name, percent, total_seconds:total}) => ({name, percent:percent / 100, total})).sort((a, b) => b.percent - a.percent)
+    if (!projectStats)
+      return undefined
```

Comment:
It's just `projects` that should be undefined when `projectStats` is, but the plugin is still expected to return a result in any case

Reply:
> It's just `projects` that should be undefined when `projectStats` is, but the plugin is still expected to return a result in any case

Oh, I'm sorry, I misunderstood the previous feedback.